transport/transport-runtime/src/androidTest/java/com/google/android/datatransport/runtime/scheduling/persistence/SpyEventStoreModule.java

```diff
@@ -39,8 +40,9 @@ static SQLiteEventStore sqliteEventStore(
       @WallTime Clock wallClock,
       @Monotonic Clock clock,
       EventStoreConfig config,
-      SchemaManager schemaManager) {
-    return spy(new SQLiteEventStore(wallClock, clock, config, schemaManager));
+      SchemaManager schemaManager,
+      @Named("PACKAGE_NAME") String packageName) {
```

Comment:
Let's use a custom annotation instead of `@Named`, to quote [this page](https://github.com/google/guice/wiki/BindingAnnotations#named):

> Since the compiler can't check the string, we recommend using `@Named` sparingly. Defining your own purpose-built annotations provides better type-safety.

Here's an example of such an annotation: https://github.com/firebase/firebase-android-sdk/blob/7c1afeb999ba1452f6d38014cb394ee3d44f2bc4/transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/time/Monotonic.java#L19-L20

Reply:
The reason i choose `@Named` is that `EventSoreModuule` used `@Named("SQLITE_DB_NAME")` to provide `dbName`, just trying to be consist with it.

Reply:
ok, sgtm then